cmd/print.go

```diff
@@ -176,6 +179,9 @@ func newPrintCommand() *cobra.Command {
 		`If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even
 if specified with --namespace.`)
 
+	cmd.Flags().StringSliceVarP(&pr.providers, "providers", "p", i2gw.GetSupportedProviders(),
+		`Use these providers specified by the option to execute convert actions`)
```

Comment:
maybe `fmt.Sprintf("If present, the tool will try to convert only resources related to the specified providers, supported values are %v", i2gw.GetSupportedProviders())`